src/main/utilities/test/src/ModelImportTests.cpp

```diff
@@ -22,12 +22,16 @@ using std::endl;
 class ModelImportTest: public ::testing::Test {
  protected:
     void SetUp() override {
-        auto dummyController = DummyGfxController();
-        modelImport = new ModelImport("dummy", texturePathStage, texturePatternStage, 0, &dummyController);
+        dummyController = new DummyGfxController();
```

Comment:
Really don't understand why this didn't cause issues on Linux... The DummyGfxController is destroyed as soon as the SetUp method ends (scope lifetime). This should have cause segfaults on Linux too, but it didn't... It's fixed now